Dockerfile

```diff
@@ -20,22 +21,6 @@ COPY vite.config.ts ./
 # Copy the source code to the working directory
 COPY src ./src
 COPY static ./static
-# COPY .svelte-kit ./.svelte-kit
```

Comment:
I think you need to keep/ uncomment this part out

Reply:
Ah, it was commented out on `main`, so I thought it was good to drop. Yeah, that eliminates that error.

Reply:
Can you also copy `.svelte-kit` into the repo, or is that something a user is supposed to create themselves?

Reply:
I don't know enough about Svelte to know where it comes from My working copy has the directory, but when I do a fresh clone, it doesn't. 🤔

Reply:
Oh! if I remove it, then do `npm run build`, it throws the warning about the missing file, but then the directory is there.

```
▲ [WARNING] Cannot find base config file "./.svelte-kit/tsconfig.json" [tsconfig.json]
```

Reply:
actually, if I leave out `COPY .svelte-kit ./.svelte-kit` (otherwise it can't build because that will always fail), when I run `docker compose up --build`, it appears to create `.svelte-kit` and create files in there on its own 🤷🏻

Reply:
and it works and I can reach it at http://localhost:3000 ❤️

Reply:
I think I got it sorted in https://github.com/RMI-PACTA/pacta-dashboard-svelte/pull/109/commits/a7d558ee24e829479505f3993c8c4ebb8458c719.

The `.svelte-kit` directory is created _by svelte_, and needs to exist prior to `npm run build`, so the suggestion is to create a `prepare` step.

[This comment](https://github.com/sveltejs/kit/issues/5390#issuecomment-1176480653) pointed me in the right direction.

Reply:
ironically, now I also see the `Cannot find base config file "./svelte-kit/tsconfig.json` warning, but it seems like the `./svelte-kit/` directory is eventually created and populated, and the server spins up and is accessible 🤷🏻